### PR TITLE
Ignore ActionMarkAsRead when removing tag from entry

### PR DESCRIPTION
**Pull request by Kdecherf**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | somewhat?
| Deprecations? | no
| Tests pass?   | -
| Documentation | no
| Translation   | no
| Fixed tickets | #2835 
| License       | MIT

Ignore ActionMarkAsRead to stay on entry when removing a tag.

### CODE REVIEW

**Comment by Kdecherf** [comment]
yep, i'll work on the test and php-cs-fixer next week

**Comment by Kdecherf** [comment]
I've added two new tests and updated one existing (but this last one is failing). I also fixed files with php-cs-fixer.

File: tests/Wallabag/CoreBundle/Controller/TagControllerTest.php [hunk 1]
```diff
@@ -126,9 +126,13 @@ public function testRemoveTagFromEntry()
             ->getRepository('WallabagCoreBundle:Tag')
             ->findOneByEntryAndTagLabel($entry, $this->tagName);
 
+        // We make a first request to set an history and test redirection after tag deletion
+        $client->request('GET', '/view/'.$entry->getId());
```
Comment (Kdecherf) [inline tests/Wallabag/CoreBundle/Controller/TagControllerTest.php]: @j0k3r @nicosomb this new request is making the next `assertNotContains` call to fail.

It seems that `$entry` is not correctly updated when accessed during assertion. Do you have any idea?

Comment (j0k3r) [inline tests/Wallabag/CoreBundle/Controller/TagControllerTest.php]: This should be fixed.
I don't know why it worked before...
I just re-retrieved the entry to be sure to then fetch tags correctly, and it works.

**Pull request merged**